src/main/java/dk/sdu/se_f22/productmodule/management/IProductManager.java

```diff
@@ -17,5 +17,4 @@
     void reparse();
 
     ArrayList<Product> getAllProducts();
-
```

Comment:
I'm not a fan of changing files in other modules.
Unless given approval by the group, it can be confusing for some etc.

Reply:
Understandable, will resolve